.github/workflows/trivy.yaml

```
@@ -79,11 +80,24 @@ jobs:
               headers: {
                 'X-GitHub-Api-Version': '2022-11-28'
               }
-            })            
-            return {
-                  git_tag: packages[0].metadata.container.tags[0].split("-")[0],
-                  container_tag: packages[0].metadata.container.tags[0]
+            })
+            const tagRegex = /^[0-9]+.[0-9]+.[0-9]+\-/;
+            const matchingTags = [];
+            for (const pkg of packages) {
+              const tags = pkg.metadata?.container?.tags || [];
```

nit: Are these expected to be falsy or null/undefined? If it's the latter, we must use the nullish operator:
```suggestion
              const tags = pkg.metadata?.container?.tags ?? [];
```

---

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

---

My fear sparks from:
> The optional chaining (?.) operator accesses an object's property or calls a function. If the object accessed or function called using this operator is [undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined) or [null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/null), the expression short circuits and evaluates to [undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined) instead of throwing an error.